cluster-up/cluster/ocp-4.3/provider.sh

```diff
@@ -51,6 +51,9 @@ function up() {
         params=" --container-registry= $params"
     fi
 
+    # Free some cached/buffered mem
+    sync; echo 3 > /proc/sys/vm/drop_caches
```

Comment:
This is not related to OCP-4.4. If you need to do a fix here, please elaborate in the commit message.